canonical/canonical.com
# WD-12243 Dev Create data/mongodb/what-is-mongodb

## Done

Created data/mongodb/what-is-mongodb
Assets are still to be updated
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Perform the checks for the new-webpage.
- [Design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=206-2888&t=PfDFMvTJxAM5v95v-0)
- [Copydoc](https://docs.google.com/document/d/1hET-ylqXq96mOfj4hovqilX5Nl0J8FLw1gkRMJkIjp8/edit#heading=h.gjjixnphncvc)
- [Demo](https://canonical-com-1343.demos.haus/data/mongodb/what-is-mongodb)

## Issue / Card

Fixes # [WD-12243](https://warthogs.atlassian.net/browse/WD-12243)

## Screenshots

[if relevant, include a screenshot]


[WD-12243]: https://warthogs.atlassian.net/browse/WD-12243?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
